vyper/semantics/types/indexable/sequence.py

```diff
@@ -100,6 +100,9 @@ def is_dynamic_size(self):
     def size_in_bytes(self):
         return self.value_type.size_in_bytes * self.length
 
+    def get_index_type(self, node):
+        return self.key_type
```

Comment:
does `key_type` exist on sequences?

Reply:
yes, it does. they are of `IntegerAbstractType`.

Reply:
of course it does :sweat_smile: